### PR TITLE
Use former pool values in multi-pool config

### LINKED ISSUE #329: Backwards compatibility broken
The recently merged PR https://github.com/geerlingguy/ansible-role-php/pull/302 broke backwards compatibility. FPM configuration items for pool settings have been renamed. If you have these configured they no longer have any effect and default config will be used:

```yaml
php_fpm_listen
php_fpm_listen_allowed_clients
php_fpm_pm_max_children
php_fpm_pm_start_servers
php_fpm_pm_min_spare_servers
php_fpm_pm_max_spare_servers
```

### PR DESCRIPTION
**Pull request by shakalandy**
make #302 backwards compatible, fixes #329
based on comment from https://github.com/geerlingguy/ansible-role-php/pull/302#issuecomment-741609370

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -24,19 +24,24 @@ php_enable_php_fpm: false
...
-    pool_listen: "127.0.0.1:9000"
-    pool_listen_allowed_clients: "127.0.0.1"
+    pool_listen: "{{ php_fpm_listen }}"
+    pool_listen_allowed_clients: "{{ php_fpm_listen_allowed_clients }}"
     pool_pm: dynamic
-    pool_pm_max_children: 5
-    pool_pm_start_servers: 2
-    pool_pm_min_spare_servers: 1
-    pool_pm_max_spare_servers: 3
-    pool_pm_max_requests: 500
```
Comment (tjoelsson) [inline defaults/main.yml:39]: I think this line should be kept

Comment (shakalandy) [reply defaults/main.yml:39]: Readded with 5e4b654
Somehow went through

Comment (tjoelsson) [reply defaults/main.yml:39]: Actually, the default value is 0. Maybe the role should have the same default.

https://www.php.net/manual/en/install.fpm.configuration.php

Comment (shakalandy) [reply defaults/main.yml:39]: You're right, set in 170c409

**Comment by geerlingguy** [comment]
Merged and released bugfix 4.6.1 release.

**Comment by shakalandy** [comment]
@geerlingguy  thanks for merging!
Build failed on debian - can't see the error on that?

**Pull request merged**